### ISSUE TITLE
FIPS automation with 4.14 fails

### ISSUE THREAD

**Issue by fbalak** [red-hat-storage/ocs-ci#8278]
Deployment of fips cluster (FIPS checkbox checked in jenkins) fails on OCP 4.14 with:
```ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n openshift-storage exec prometheus-adapter-6cb87d55b8-t9clf -- bash -c "cat /proc/sys/crypto/fips_enabled".
Error is time="2023-08-16T11:39:01Z" level=error msg="exec failed: unable to start container process: exec: \"bash\": executable file not found in $PATH"
command terminated with exit code 255
```
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/28225/testReport/junit/tests.ecosystem.deployment/test_deployment/test_deployment/

**Comment by vavuthu** [comment]
this is fixed in 4.14.0-110 build.

**Comment by petr-balogh** [comment]
We also had a bug for it:
https://bugzilla.redhat.com/show_bug.cgi?id=2228805
